lib/src/elastic_client_impl.dart

```diff
@@ -207,3 +207,9 @@ class ClearScrollResult {
   Map<String, dynamic> toMap() =>
       {'succeeded': succeeded, 'numFreed': numFreed};
 }
+
+class TermEnumResult {
+  final List<String> terms;
+
+  TermEnumResult(this.terms);
+}
```

Comment:
Add extra newline

Reply:
I am curious, can you please explain why adding a new line at the end of the file?

Reply:
It's just coding style consistency.